integrationtests/scenarios/i002335-razorengine/before/MyClassLibrary/MyClassLibrary/MyClassLibrary.csprojtemplate

```diff
@@ -501,9 +492,23 @@
         </Reference>
       </ItemGroup>
     </When>
+    <When Condition="'$(RAZOR4)' == 'True' And (($(TargetFrameworkIdentifier) == '.NETStandard' And ($(TargetFrameworkVersion) == 'v1.6' Or $(TargetFrameworkVersion) == 'v2.0')) Or ($(TargetFrameworkIdentifier) == '.NETCoreApp' And ($(TargetFrameworkVersion) == 'v1.0' Or $(TargetFrameworkVersion) == 'v1.1' Or $(TargetFrameworkVersion) == 'v2.0')))">
+      <ItemGroup>
+        <Reference Include="System.AppContext">
+          <HintPath>..\..\packages\razor4\System.AppContext\ref\netstandard1.6\System.AppContext.dll</HintPath>
+          <Private>False</Private>
+          <Paket>True</Paket>
+        </Reference>
+        <Reference Include="System.AppContext">
+          <HintPath>..\..\packages\razor4\System.AppContext\ref\netstandard1.6\System.AppContext.xml</HintPath>
+          <Private>False</Private>
+          <Paket>True</Paket>
+        </Reference>
```

Comment:
This is another bug I'm fixing another time https://github.com/fsprojects/Paket/issues/2390